spec/support/geckoboard_helpers.rb

```diff
@@ -2,9 +2,9 @@ module GeckoboardHelpers
   # Use this to stub out the API calls that will be made to Geckoboard when a
   # Claim is submitted. It will return the webmock POST request, which can be
   # used to write expectations about the request being made.
-  def stub_geckoboard_submitted_event
-    stub_geckoboard_dataset_find_or_create("claims.submitted.test")
-    stub_geckoboard_dataset_post("claims.submitted.test")
+  def stub_geckoboard_submitted_event(dataset_id = "claims.submitted.test")
```

Comment:
The comment is outdated here, and I think that the method name might be too (although I'm not sure if `submitted` in the method name refers to a claim being submitted or data being submitted to Geckoboard).